.github/workflows/build.yaml

```diff
@@ -49,15 +52,16 @@ jobs:
                     - description: 'Lowest deps'
                       php: '8.1'
                       composer_option: '--prefer-lowest'
-                      env:
-                          SYMFONY_DEPRECATIONS_HELPER: max[self]=0
                     - description: 'Symfony 5.4'
                       php: '8.1'
                       symfony: 5.4.*
-                    - description: 'Dev deps'
+                    - description: 'Symfony 6.4'
                       php: '8.2'
+                      symfony: 6.4.*
+                    - description: 'Dev deps'
+                      php: '8.3'
                       dev: true
-                      symfony: 7.0.*
+                      symfony: 7.1.*
```

Comment:
the job using dev deps should not force a Symfony version at all (especially given that this is *not* the dev version of Symfony)